src/metrics.jl

```diff
@@ -208,46 +208,46 @@ kl_divergence(a::AbstractArray, b::AbstractArray) = evaluate(KLDivergence(), a,
 
 # RenyiDivergence
 function eval_start{T<:AbstractFloat}(::RenyiDivergence, a::AbstractArray{T}, b::AbstractArray{T})
-    zero(T), zero(T)
+    zero(T), zero(T), sum(a), sum(b)
```

Comment:
@KristofferC This is because of accumulation errors in `Float64`s - they turn out to be strewn throughout these distance measures unfortunately. sum(a) is often 0.99999999999 or so, and this causes v slight negative distances in some places, and other similar errors. I fixed them here for Renyi, but I don't know think there's a general way of doing it for other metrics, particularly when they are not probability distributions, so they shouldn't sum to 1 anyway.